packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -652,6 +667,7 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {
 				if (selectedButton) {
 					selectedButton.blur();
 					selectedButton.focus();
+					Spotlight.focus(this.props.moreButtonSpotlightId);
```

Comment:
@germboy: I may need your help on this. This seems a bit odd to me.
If the current spotted item is not the one that needs to be refocused, then I'd like to focus on `moreButton` determined by `moreButtonSpotlightId`. Hence this line.
The reason why I couldn't remove the `focus` call above is that if the previously focused component is the same, then highlight will not be restored when clicked with the pointer. (It's okay in 5-way)

Is there another way to properly set "(re)focus" to a component?